app/views/brexit_checker/_results_citizen_actions.html.erb

```diff
@@ -0,0 +1,31 @@
+<% heading = t('brexit_checker.results.audiences.citizen.heading') %>
```

Comment:
using the translation directly is probably easier to read

Reply:
even more so here, it's go the same two re-uses that the biz layer has plus we're already in a bit of string interpolation hell on line 22.

Seemed worth it to keep things DRY?